app/controllers/websites_controller.rb

```diff
@@ -1,30 +1,7 @@
 class WebsitesController < ApplicationController
-
   def create
-    @website = current_user.websites.build(website_params)
-    if @website.save
-      @sites_count=Website.count('id')
-      @site_name=[]
-      current_user.websites.each do |website|
-          @site_name << website.url
-      end
-      
-      @each_user_sites=current_user.websites.count
-      if User.exists?
-        current_user.update(sites: @site_name)
-        current_user.update(sites_number: @each_user_sites)
-      else
-        current_user.create(sites: @site_name)
-        current_user.create(sites_number: @each_user_sites)
-      end
-      if Statistic.exists?
-        Statistic.update(sites: @sites_count) 
-      else
-        Statistic.create(sites: @sites_count)
-      end
-    end
-    FetchRankJob.perform_later(@website) 
-    redirect_to root_url
+   @descript = Websites::Create.call(params, current_user)
```

Comment:
Isn't `description` an attribute of `Website`? If so, return the website itself and use the attribute here.

When somebody sees the code, they wouldn't expect `Websites::Create.call(...)` to return some arbitrary attribute of the model, but the model itself. So it's always preferable to return the `Website` object, rather than a string.